hostfactory/host_provider/generateWeightedTemplates.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+export HF_LOGDIR=/tmp/log
+export HF_CONFDIR=$HF_TOP/conf
+export HF_WORKDIR=/tmp/work
+mkdir -p $HF_LOGDIR
+mkdir -p $HF_WORKDIR
+cat <<EOF >/tmp/input.json
```

Comment:
We should probably pick something a bit more unique than input.json - i.e. generateWeightedTemplates.input.json or something